File: DSA/Graph/bfs.py
from Queue import queue
from dictionary import g1
def bfs(q,vertex,visited):
    q.enqueue(vertex)
    while not q.isEmpty():
        element=q.dequeue()
        if element not in visited:
            visited.append(element)
        for value in g1.list[element]:
            if value not in visited:
                q.enqueue(value)
q=queue()
visited=[]
bfs(q,"A",visited)
print(visited)

File: DSA/Graph/bfs.java
import java.util.*;
public class bfs{
    public static void bfs(Queue queue){
        int length=queue.length();
        System.out.println(length);
    }
    public static void main(String args[]){
        Queue<Integer> queue = new PriorityQueue<Integer> (); 
        queue.add(1);
        queue.add(2);
        queue.add(4);
        bfs();

    

    }
}